### PR TITLE
PE-2784: storage permissions for android 13

### PR DESCRIPTION
**Pull request by karlprieb**

### CODE REVIEW

File: lib/src/utils/permissions.dart [hunk 1]
```diff
@@ -60,3 +65,14 @@ Future<T> secureScopedAction<T>(
 
   return value;
 }
+
+Future<bool> isAndroidSdkBiggerThan33() async {
```
Comment (matibat) [inline lib/src/utils/permissions.dart:69]: Nit: maybe "isAndroidSdk33OrGreater", the version 33 would also make the method return true

**Review by matibat — APPROVED** [review]
Approved with a nit

File: lib/src/utils/permissions.dart (hunk 1 above)
Comment (karlprieb) [reply lib/src/utils/permissions.dart:69]: that’s a better name for sure :)

Comment (karlprieb) [reply lib/src/utils/permissions.dart]: 545e620099dc5702ac423b672d88c0c912b89699

**Review by matibat — APPROVED** [review]
Thanks! 🚀

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM! And I'm very happy to see this package in its own repo! thanks!

**Pull request merged**